lambdas/meetup/config.js

```diff
@@ -0,0 +1,32 @@
+const convert = params =>
+  Object.entries(params)
+    .map(([key, val]) => `${key}=${val}`)
+    .join("&");
+
+const groupsApi = "https://api.meetup.com/find/groups?";
+const groupsParams = convert({
+  "photo-host": "public",
+  country: "GB",
+  lon: -6.3133593,
+  lat: 54.8632911, // Ballymena
+  radius: 50, // 50 mile radius (most of Northern Ireland)
```

Comment:
Actually, with the country restriction above, this might be tweak-able to cover all of NI - having a 60 mile radius of somewhere central (cookstown?) would completely cover NI, with Irish meetups being filtered out by the country restriction.